src/Synapse/Mapper/FinderTrait.php

```diff
@@ -4,7 +4,7 @@
 
 use InvalidArgumentException;
 use LogicException;
-use Synapse\Mapper\PaginationData;
```

Comment:
PaginationData is in the same namespace as the FinderTrait, therefore no use statement is needed.  AbstractEntity is not in the same namespace.